tests/api_test/api_test.cpp

```diff
@@ -1104,7 +1108,7 @@ _load_invalid_program(_In_z_ const char* file_name, ebpf_execution_type_t execut
 
 TEST_CASE("load_native_program_invalid1", "[native][negative]")
 {
-    _load_invalid_program("invalid_maps1.sys", EBPF_EXECUTION_NATIVE, -EINVAL);
+    _load_invalid_program("invalid_maps1.sys", EBPF_EXECUTION_NATIVE, 0);
```

Comment:
question: Does this mean loading of invalid program should succeed?

Reply:
No, it means loading of a file with 0 programs in it should succeed.  There is no invalid program in invalid_maps1.sys.

Reply:
Due to 0 programs, with new changes, I think it may be skipping the load of native module altogether.

Reply:
Basically, here the API is wrongly returning SUCCESS even though the module has not been loaded, and the maps have also not been created.

Reply:
Updated to create maps.